src/lib/openapi/spec/admin-features-query-schema.ts

```diff
@@ -4,18 +4,20 @@ export const adminFeaturesQuerySchema = {
     $id: '#/components/schemas/adminFeaturesQuerySchema',
     type: 'object',
     additionalProperties: true,
+    description: 'Used to filter feature toggles from the admin-api',
```

Comment:
Sorry, I was a bit quick here. I thought this was a data object and not query parameters. The query params object doesn't accept a top-level description (from what I can tell based on [the query params description from swagger.io](https://swagger.io/docs/specification/describing-parameters/#query-parameters). 

```suggestion
```